lib/utils.js

```diff
@@ -21,3 +26,41 @@ export function defer () {
     reject: rej
   }
 }
+
+/**
+ * Returns search paths for the webtransport.node binary in
+ * order - callers should load the first path that exists
+ *
+ * @param {boolean} [includeDebug]
+ * @returns
+ */
+export function findWTBinaryPaths (includeDebug) {
+  const dirname = url.fileURLToPath(new URL('..', import.meta.url))
+
+  /** @type {string[]} */
+  const wtpaths = [
+    `${dirname}build/Release/webtransport.node`, // precompiled version
```

Comment:
I am not sure. but I would prefer that the local build version always wins. It looks like in this case the precompiled always wins.

Reply:
That was the intention, to favour the happy path of users who have installed the module on a platform that has a precompiled version available, rather than developers who will not have the precompiled version present.

Reply:
Ok, but you have to trigger a build manually. So the happy folks would never be bothered, only if they trigger a build. 
So maybe switching the lines and adding a warning, if the local build is used?